mscore/seq.cpp

```diff
@@ -833,8 +833,10 @@ void Seq::process(unsigned framesPerPeriod, float* buffer)
                         if (mscore->loop()) {
                               int loopOutUTick = cs->repeatList().tick2utick(cs->loopOutTick().ticks());
                               if (loopOutUTick < scoreEndUTick) {
-                                    // Also make sure we are not "before" the loop
-                                    if (playPosUTick >= loopOutUTick || cs->repeatList().utick2tick(playPosUTick) < cs->loopInTick().ticks()) {
+                                    qreal framesPerPeriodInTime = static_cast<qreal>(framesPerPeriod) / MScore::sampleRate;
+                                    signed framesPerPeriodInTicks = cs->utime2utick(framesPerPeriodInTime);
```

Comment:
The MuseScore codebase doesn't use `signed` for `int`. This should be either `int` or `auto`.

Reply:
PR has been updated.